### PR TITLE
zh: some mirror updates

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- update some links to MySQL docs because `slave` has been updated
- add more explanation about HA
- add `max-allowed-packet` into task config sample
- mirror updates the format about TiUP doc

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: zh/task-configuration-file-full.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ target-database:                # 下游数据库实例配置
   port: 4000
   user: "root"
   password: "/Q7B9DizNLLTTfiZHv9WoEAKamfpIUs="  # 推荐使用经 dmctl 加密后的密码
+  max-allowed-packet: 67108864                  # 设置 DM 内部连接 TiDB 时客户端的 "max_allowed_packet" 限制，单位为字节，默认 67108864 (64 MB)
```
Comment (csuzhangxc) [inline zh/task-configuration-file-full.md:45]: this can be removed after implemented https://github.com/pingcap/dm/issues/1003.

**Comment by csuzhangxc** [comment]
@lance6716 @ireneontheway PTAL

**Comment by lance6716** [comment]
LGTM

File: zh/task-configuration-file-full.md (hunk 1 above)
Comment (yikeke) [inline zh/task-configuration-file-full.md:45]: `设置 DM 内部连接 TiDB 时客户端的 "max_allowed_packet" 限制`：这句话能否用中文解释 "max_allowed_packet"？（不要用  "max_allowed_packet"  来解释  "max_allowed_packet"）

Comment (csuzhangxc) [reply zh/task-configuration-file-full.md:45]: `max_allowed_packet` 是 MySQL/TiDB 里的一个变量

Comment (yikeke) [reply zh/task-configuration-file-full.md:45]:
```suggestion
  max-allowed-packet: 67108864                  # 设置 DM 内部连接 TiDB 服务器时，TiDB 客户端接受的数据包大小限制，单位为字节，默认 67108864 (64 MB)
```
查了下，这样解释准确不 ⬆️

Comment (csuzhangxc) [reply zh/task-configuration-file-full.md:45]: 解析了 `max_allowed_packet` 后，会不会用户又和这个 TiDB 内的这个变量对应不上了？

Comment (yikeke) [reply zh/task-configuration-file-full.md:45]: 改了下，再看看呢

Comment (csuzhangxc) [reply zh/task-configuration-file-full.md]: LGTM

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**